packages/n/nftables/package.yml

```diff
@@ -14,23 +16,34 @@ builddeps  :
     - pkgconfig(libmnl)
     - pkgconfig(libnftnl)
     - pkgconfig(python3)
-    - pkgconfig(xtables)
     - docbook2x
+    - python-build
+    - python-installer
+    - python-wheel
+rundeps    :
+    - ^python-nftables:
+        - nftables
+clang      : yes
+optimize   : thin-lto
 setup      : |
-    %patch -p1 -i $pkgfiles/0001-Replace-distutils-with-setuptools.patch
-    %configure \
-        --disable-static \
-        --with-json \
-        --disable-debug \
-        --with-cli=readline \
-        --with-python-bin=/usr/bin/python3
+    %patch -p1 -i $pkgfiles/0001-Stateless.patch
+    %reconfigure \
+                 --disable-debug \
+                 --disable-python \
```

Comment:
You're the expert here, but why are you disabling python, yet shipping python-nftables...? Is it to do with disabling being able to programmatically interface with nftables via a python C module?

Reply:
We're disabling python as part of the main build, but building it later with the `python3 -m build --wheel` command.